storybook/pages/StatusButtonRowPage.qml

```diff
@@ -0,0 +1,71 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+
+import utils 1.0
+
+import Storybook 1.0
+
+SplitView {
+    orientation: Qt.Horizontal
+
+    Logs { id: logs }
+
+    QtObject {
+        id: d
+        readonly property var values: [0.1, 0.5, 0.7, 1] // predefined values
+    }
+
+    ListModel {
```

Comment:
it's ok ofc, but I would make it even simpler, just
```
    ListModel {
        id: valuesModel

        ListElement { value: 1; text: "A" }
        ListElement { value: 2; text: "B" }
        ListElement { value: 3; text: "C" }
        ListElement { value: 4; text: "D" }
    }
```
no `d` object and `Component.onCompleted` then

Reply:
In the end I decided to go with numbers, just to show how to fill the custom model. Ofc, the easier (and obvious imho) would be to go with a `ListModel`